plugins/hooks/routes/crash.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Boom = require('boom');
+const Joi = require('joi');
+
+const controllerFunc = require('../controller');
+
+module.exports = {
+  path: '/v1/hooks/crash',
+  method: 'POST',
+  config: {
+    description: 'Webhook endpoint for the crash event',
+    tags: ['api', 'v1', 'crash'],
+    response: {
+      schema: Joi.object({ status: 'OK' })
+    }
+  },
+  handler: (req, reply) => {
+    const caller = req.payload.project.domain;
+    const authClient = req.server.plugins['authentic-client'].client;
+    const controller = controllerFunc(authClient, req);
+    const principalId = req.payload.project.name;
+    const keyId = req.payload.project.id;
```

Comment:
I agree there is certainly room to improve. We will need to address the eventual need of multiple servos, but it might be slightly out of scope currently. I can add a task on our backlog and we can pull it in when we think it's necessary.